web/router/admin.go

```diff
@@ -15,8 +20,12 @@ func Admin(m *mux.Router) *mux.Router {
 		m = mux.NewRouter()
 	}
 
-	m.Path("/").Methods("GET").Name(AdminDashboard)
-	// m.Path("/settings").Methods("GET").Name(AdminSettings)
+	m.Path("/dashboard").Methods("GET").Name(AdminDashboard)
+
+	m.Path("/allow-list").Methods("GET").Name(AdminAllowListOverview)
```

Comment:
The name allow-list is totally fine for implementation, it can stay like that. I was thinking that for user-facing situations, we would call this the "members" list. Like when someone gets added to a room, they become a room *member*. URLs are not the most visible "user interface" but it would make it easier for users to understand what the URL is if it were set to `/members`, `/members/add`, `/members/confirm`, `/members/remove`.

What do you think about changing these URLs here, while keeping the rest of the code using the name AllowList?

Reply:
> What do you think about changing these URLs here, while keeping the rest of the code using the name AllowList?

Yup, right. Agreed. Do we have a candidate for the denied/blocked ones?

Reply:
Those we be called `banned` (and the verb, "to ban"), to distinguish banning from blocking. Banning is about rejecting connections (either IP-based or ID-based or both).